pkg/metrics/tags.go

```diff
@@ -58,6 +58,7 @@ var (
 	allowedEventTypes = map[string]struct{}{
 		"e2e-dummy-event-type":              {},
 		"e2e-testing-resp-event-type-dummy": {},
+		"_invalid_cloud_event_": {},
```

Comment:
The alignment of `{}` suggests that your editor is not set up with auto formatting properly. Make sure you either set it up or add a commit hook to your git repo.